web/temp_views.py

```
@@ -0,0 +1,54 @@
+@login_required
+def get_update_round_html(request, classroom_id):
+    """Get the HTML for the update round container."""
+    classroom = get_object_or_404(VirtualClassroom, id=classroom_id)
```

Ruff (0.8.2)

Undefined name `get_object_or_404`

(F821)